builtInFunctions/multiESDTNFTTransfer.go

```diff
@@ -201,6 +204,15 @@ func (e *esdtNFTMultiTransfer) ProcessBuiltinFunction(
 		} else {
 			transferredValue := big.NewInt(0).SetBytes(vmInput.Arguments[tokenStartIndex+2])
 			value.Set(transferredValue)
+
+			if bytes.Equal(e.baseTokenID, tokenID) {
+				err = acntDst.AddToBalance(transferredValue)
```

Comment:
To prevent the problem of sending EGLD like this to systems that do not support ESDT data parsing, can we make sure of 2 things?
1. we allow EGLD sent as ESDT only on esdtNFTMultiTransfer operation, and
2. the esdtNFTMultiTransfer contains at least 2 different tokens.

Reply:
First point is implemented. I would not want to implement the second one, as it would need ifs/elses in rust framework.